app/src/js/components/DropDown/dropdown.js

```diff
@@ -1,143 +1,146 @@
 'use strict';
-import c from 'classnames';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router-dom';
-import Autocomplete from 'react-autocomplete';
-import { initialValueFromLocation } from '../../utils/url-helper';
 import withQueryParams from 'react-router-query-params';
-
-function shouldItemRender ({ label }, value) {
-  return label.toLowerCase().indexOf(value) >= 0;
-}
-
-function renderItem (item, highlighted) {
-  return <div className={c('autocomplete__select', {
-    'autocomplete__select--highlighted': highlighted
-  })} key={item.value}>{item.label}</div>;
-}
-
-function renderMenu (items, value, style) {
+import {
+  Typeahead,
+  Menu,
+  MenuItem,
+  Hint,
+  Input,
+} from 'react-bootstrap-typeahead';
+
+function renderInput({ inputRef, referenceElementRef, ...inputProps }) {
   return (
-    <div className='autocomplete__menu' style={style} children={items} />
+    <Hint
+      shouldSelect={(shouldSelect, e) => {
+        // Selects the hint when the user hits the 'enter' key.
+        return e.keyCode === 13 || shouldSelect;
```

Comment:
When I type a number and enter, it doesn't work, until I select the number I typed in.

Reply:
This was a little more involved because when adding a new option, there is no `<Hint>` component. That's why it wasn't working on enter in that case. 

As a workaround, I added an `onKeyDown` handler that fires when `allowNew === true` and `e.keyCode === 13`. It takes `e.target.value` and makes that the selection. Also had to add a ref to the <Typeahead> component so that the menu could be closed after this was called.

https://github.com/nasa/cumulus-dashboard/pull/812/commits/6d6cc851719f7ce83475edb7bdf1e47be511a4ae